lib/ecom/session.ts

```diff
@@ -12,7 +12,14 @@ export type SessionData = {
     wixClientId: string;
 };
 
-const { getSession, commitSession } = createCookieSessionStorage<SessionData, void>({
+type FlashData = {
```

Comment:
why `Flash`?

Reply:
I see... the name defined in the `createCookieSessionStorage` 🤔

Reply:
yes. you can dafine data and flash data for session. data - persisted for a long time. flash data - persisted until first read